docs/source/data/subscriptions.mdx

```diff
@@ -676,5 +680,65 @@ Use [`WebSocketLink`](/react/api/link/apollo-link-ws/)
 </tbody>
 </table>
 
-For more information on using Apollo Server with `subscriptions-transport-ws`, you can view the [previous version of this article here](https://github.com/apollographql/apollo-server/blob/4b82967c10a69aa104848a8c9d15c591b528918c/docs/source/data/subscriptions.mdx).
+### Switching to `graphql-ws`
+
+> The following steps assume you've already [swapped to the `apollo-server-express` package](../integrations/middleware/#swapping-out-apollo-server).
+
+To switch from `subscriptions-transport-ws` to `graphql-ws`, you can follow the steps below.
+
+1. Install `graphql-ws` and `ws` into your app:
```

Comment:
It feels like this whole section is very similar to the "enabling subscriptions" section? Maybe it would be more helpful if it showed s-t-w and g-w examples side by side?

I also think it would be helpful to show how you port code that sets the operation context from s-t-w to g-w. (Not ultra obvious, esp since the s-t-w function isn't named context!)

Reply:
Great idea! I added some Code columns for the basic s-t-w => g-w code, and an extra section calling out how how to set up GraphQL operation context for `graphql-ws`.